projects/aca-content/src/lib/components/search/search-input/search-input.component.ts

```diff
@@ -140,7 +152,6 @@ export class SearchInputComponent implements OnInit, OnDestroy {
     } else {
       this.store.dispatch(new SnackbarErrorAction('APP.BROWSE.SEARCH.EMPTY_SEARCH'));
     }
-    this.trigger.closeMenu();
```

Comment:
Why did you remove this?

Reply:
this is strange, was thinking about the same, note that I'm just cleaning the garbage here :) will restore